.sphinx/_static/github_issue_links.js

```diff
@@ -0,0 +1,26 @@
+window.onload = function() {
+    const link = document.createElement("a");
+    link.classList.add("muted-link");
+    link.classList.add("github-issue-link");
+    link.text = "Have a question?";
```

Comment:
As this button causes something to happen, I would like the text to reflect that - so It should be action text, e.g. "Ask us a question or leave feedback"

Reply:
It's up to you - I won't be able to do it until Monday at the earliest though. What's the exact text we want to have on the button though? Is it "ask us a question" or "leave feedback"? (having both will make the text far too long for a button - it should be as short as possible to make it easier to read)

Reply:
No hurry, I just don't want it to get lost. :)
I would vote for "Open an issue", since that's what the button does. But that might be too technical?

Reply:
I changed it to "give feedback", which hopefully should suit our purposes.